pycuda/gpuarray.py

```diff
@@ -1767,6 +1767,104 @@ def make_func_for_chunk_size(chunk_size):
 
 # {{{ shape manipulation
 
+def concatenate(arrays, axis=0, allocator=None):
+
+    # impllementation is borrowed from pyopencl.array.concatenate()
```

Comment:
- Typo.
- Should come after docstring.

Reply:
Thanks for finding the typo. The typo has been fixed and the acknowledgment as well fold marker have been relocated.